honeybee_energy/ventcool/crack_data.py

```diff
@@ -0,0 +1,89 @@
+# coding':utf-8
+"""Crack data dictionary.
+
+This data is obtained from the DesignBuilder Crack Template. Note that this
+crack data is normalized by area and perimeter for face cracks and subface edges,
+respectively. In EP face cracks are not normalized by area, but subface edges
+are normalized by perimeter.
+"""
```

Comment:
Can we hide this whole module by calling it `_crack_data.py`?
If there are no methods or classes in the module, then there's no reason for the user to interact with it as part of the public SDK.